### PR TITLE
Enable Quartz for HPMS Ingestion

### PR DESCRIPTION
**Pull request by nflearl**
**JIRA Tickets:**

[AB2D-2115](https://jira.cms.gov/browse/AB2D-42) - HPMS Ingestion

***Related Tickets***
 
### What Does This PR Do?

Creates a Quartz timer to invoke HPMS ingestion.

### Security Implications

- [ ] This PR adds new software dependencies
- [ ] This PR modifies or invalidates our security controls
- [ ] This PR stores or transmits data that was not stored or transmitted before

- [ ] This PR requires additional review of its security implications for other reasons

### Submitter Checklist

- [x] This PR includes any required documentation changes, (`README`, changelog / release notes, website).
- [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments.
- [x] Code checked for PHI/PII exposure

### CODE REVIEW

**Review by wnyffenegger — APPROVED** [review]

**Review by lailasharshar — APPROVED** [review]

**Comment by lhanekam** [comment]
Travis CI is not working correctly, so we are using Jenkins CI to verify the build. We are waiting for approval to require Jenkins CI instead, so we are force merging as a temporary workaround.

**Pull request merged**
